### PR TITLE
fix: style prop configured as boolean in styled-system is not working

### PR DESCRIPTION
**Pull request by terryyu**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes
      /start features)

## Pull request type

Please check the type of change your PR introduces:

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Fixes: #2700

## What is the new behavior?

expand the shortcut definition by setting `property` and `scale` to the `key` name

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/styled-system/src/css.ts [hunk 1]
```diff
@@ -147,7 +147,15 @@ export const css = (args: StyleObjectOrFn = {}) => (
     const val = runIfFn(x, theme)
 
     const key = k in pseudoSelectors ? pseudoSelectors[k] : k
-    const config = (parser.config as Dict)[key]
+    let config = (parser.config as Dict)[key]
```
Comment (jvmazagao) [inline packages/styled-system/src/css.ts:150]: What's happening if the config module is false?

Comment (rr-terry-yu) [reply packages/styled-system/src/css.ts:150]: by design, the config module only takes a true value if it's meant to be translated to an config object. https://styled-system.com/custom-props#shortcut-definition
the intention for a false value needs to be defined. the current behavior for a false value is not parsing the token against the theme and output the value directly.

Comment (terryyu) [reply packages/styled-system/src/css.ts:150]: by design, the config module only takes a true value if it's meant to be translated to an config object. styled-system.com/custom-props#shortcut-definition
the intention for a false value needs to be defined. the current behavior for a false value is not parsing the token against the theme and output the value directly.

Comment (jvmazagao) [reply packages/styled-system/src/css.ts:150]: Approved.

**Review by jvmazagao — APPROVED** [review]

File: packages/styled-system/src/css.ts (hunk 1 above)
Comment (terryyu) [reply packages/styled-system/src/css.ts:150]: thanks!

**Pull request merged**